compiler/hash-typecheck/src/storage/location.rs

```diff
@@ -157,6 +213,10 @@ impl LocationStore {
             LocationTarget::Pat(pat) => {
                 self.pat_map.insert(pat, location);
             }
+            _ => {
+                // no-op
+                log::warn!("Tried to set location on a LocationTarget::Location")
```

Comment:
will this not fire on non-debug mode?

Reply:
Yes, well I think we should have logging off for release no? Otherwise what purpose is there to logging?

Reply:
Yeah im just not sure about the actual rules when it is and isn’t turned off